test/e2e/_routes.js

```diff
@@ -26,3 +26,4 @@ export const newAllocation = `${E2E.BASE_URL}/allocation/new`
 export const allocations = `${E2E.BASE_URL}/allocations`
 export const allocationsWithDate = date =>
   `${E2E.BASE_URL}/allocations/week/${date}/outgoing`
+export const singleRequests = `${E2E.BASE_URL}/moves/requested`
```

Comment:
Is this related to this PR?

Reply:
Yep. It provides the url for the single requests page so that e2e tests can check the jumper on it